### ISSUE TITLE
[TRACKER] Editor theme issues for 3.0

### ISSUE THREAD

**Issue by akien-mga** [godotengine/godot#8810]
This tracker bug is for all issues related to the new editor theme for Godot 3.0.

## PRs

- [x] New customizable editor theme #8631

## Bugs

- [x] Window dialog title bar's background are not visible on default theme #8816
- [x] New Theme: Tree drag&drop visual helper is barely visible. #8677
- [x] Script heat colors are not adjusted to new theme (they blend with background) #8608

## Enhancements

- [x] Changing the remote debugging icon #8480
- [x] Scan file selection font color broken #8635
- [x] HTML5 icon needs update #8836
- [x] Make scroll bars look different when pressed #8875
- [x] Editor Theme font color based on theme color #8893

**Comment by toger5** [comment]
Html5 icon

**Comment by akien-mga** [comment]
> Html5 icon

Hm? This is a tracker bug, please open a separate issue is something has to be done for the html5 icon.

**Comment by toger5** [comment]
I was thinking that would fit to the tracker. It was mebtioned in one discussion. Html5 icon has no highdpi and does not fit (style wise).
If you thint it doesnt, delete my messages and ill open a seperate issue (if there isnt one already)

**Comment by akien-mga** [comment]
The purpose of a tracker bug is to track links to issues on a given topic, not to discuss each of them in one giant thread. So yes, please open a separate issue for the HTML5 icon, and I'll link it here.

**Comment by n-pigeon** [comment]
I made it :P
https://github.com/godotengine/godot/issues/8836

**Comment by volzhs** [comment]
after #8631 merged

for Window dialog title bar's background are not visible on default theme #8816
![image](https://cloud.githubusercontent.com/assets/8281454/26276409/46af4262-3db2-11e7-8163-e46d480970ff.png)

for New Theme: Tree drag&drop visual helper is barely visible. #8677
![peek 2017-05-07 16-04](https://cloud.githubusercontent.com/assets/8281454/25778637/f5a7a7a6-333e-11e7-9934-060b772babde.gif)

for Script heat colors are not adjusted to new theme (they blend with background) #8608
![image](https://cloud.githubusercontent.com/assets/8281454/26276437/99abfb0e-3db2-11e7-97f0-b050af96e0db.png)

**Comment by nunodonato** [comment]
just noticed that disabled line-edits are hard to read when unfocused. 
![bugttheme](https://cloud.githubusercontent.com/assets/1642015/26331268/ef5f9e02-3f47-11e7-9170-af6c9a255481.gif)
perhaps changing the text color to something darker?

**Comment by volzhs** [comment]
@nunodonato 
![lineedit](https://cloud.githubusercontent.com/assets/8281454/26339659/6885fda4-3fc4-11e7-835f-b4279e5c6347.gif)
how about this?

**Comment by nunodonato** [comment]
damn you're fast :D

**Comment by volzhs** [comment]
I'm aware of assetlib screen issue.
beside, I'd like to know any other issue from here. :smile:

**Comment by n-pigeon** [comment]
@volzhs 
Found something. Old theme had borders and selection for active buses.
https://cloud.githubusercontent.com/assets/4701338/26264127/0286a73c-3cdc-11e7-8226-2ff903ef7c96.png

**Comment by volzhs** [comment]
@n-pigeon I tested old version.
it has border line of each bus, but not showing active one.
just first one looks different than others.
I will check it.

**Comment by toger5** [comment]
for tracker: #8893

**Comment by nunodonato** [comment]
@volzhs I noticed this now while working on the settings: the icons on the right of the controls seems like they could use a bit of padding/margin so that they dont touch the right edge

![screenshot from 2017-05-24 23-29-11](https://cloud.githubusercontent.com/assets/1642015/26428330/f6f86e22-40d8-11e7-84f0-c235568be8ab.png)

**Comment by toger5** [comment]
HTML5 icon needs update #8836
 Make scroll bars look different when pressed #8875
 Editor Theme font color based on theme color #8893

@akien-mga all of them are solved!
So another 3.0 Milestone issue closed!!

**Comment by groud** [comment]
Nice! Closing then ;)
